Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonToolkitPoweredByControl.cs

```diff
@@ -244,6 +244,11 @@ protected override void OnLoad(EventArgs e)
             base.OnLoad(e);
         }
 
+        protected override void OnCreateControl()
```

Comment:
Why is this here

Reply:
Was testing sizing. Will be cleaned up in next commit :)